core/services/bulletprooftxmanager/bulletprooftxmanager.go

```diff
@@ -503,3 +513,20 @@ func CheckEthTxQueueCapacity(db *gorm.DB, fromAddress common.Address, maxQueuedT
 	}
 	return
 }
+
+var _ TxManager = &NullTxManager{}
+
+type NullTxManager struct {
```

Comment:
Is this just easier to work with in some way vs mockery?

Reply:
It's not for tests, this is used in real code.